package.json

```diff
@@ -57,7 +57,7 @@
     "start-heroku": "tt-heroku-serve-static",
     "start-static": "tt-serve-static --config webpack.config",
     "test": "npm run jest && npm run wdio",
-    "wdio": "wdio node_modules/terra-dev-site/config/wdio/wdio.conf.js"
+    "wdio": "cross-env NODE_OPTIONS=--max-old-space-size=2048 wdio node_modules/terra-dev-site/config/wdio/wdio.conf.js"
```

Comment:
Can you use `node --max-old-space-size= 2048 `?

Reply:
Do these commands need to work on windows devices? Or are we encouraging running them from within a docker container?